src/buckets/constants/index.ts

```diff
@@ -2,7 +2,7 @@ export const isSystemBucket = (bucketName: string): boolean => {
   return bucketName.startsWith('_')
 }
 
-export const BUCKET_OVERLAY_WIDTH = 475
+export const BUCKET_OVERLAY_WIDTH = 575
```

Comment:
Is this new size intended for the feature only or even without the feature? Have you seen how this looks with the feature flag turned off?

Reply:
There's more places where this constant is used. I meant here also, where it does not have as many buttons:

BEFORE 475:
<img width="1680" alt="Screen Shot 2021-10-19 at 2 01 17 PM" src="https://user-images.githubusercontent.com/10736577/137990073-503e9e61-359f-4ea5-8700-f510a9186a4f.png">


AFTER 575:
<img width="1680" alt="Screen Shot 2021-10-19 at 2 01 56 PM" src="https://user-images.githubusercontent.com/10736577/137990107-ee32b5ca-c1e4-4928-bdd2-3e316c8271a6.png">

Reply:
Maybe we should separate this into two different constants to allow different widths in different places.